api/api/auth/tests.py

```diff
@@ -0,0 +1,144 @@
+import uuid
+from unittest import mock
+from uuid import UUID
+
+import pytest
+from django.contrib.auth import get_user_model
+from google.auth.exceptions import GoogleAuthError
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from api.auth.models import Token
+
+
+class TokenViewTest(APITestCase):
```

Comment:
@Samy-33 tests can be moved to a separate tests folder.

Reply:
Create an issue for this.